example/test_driver/purchases_flutter_e2e_test.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:async';
+import 'package:flutter_driver/flutter_driver.dart';
+
+Future<void> main() async {
+  final FlutterDriver driver = await FlutterDriver.connect();
+  final String result =
+  await driver.requestData(null, timeout: const Duration(minutes: 1));
+  await driver.close();
+  exit(result == 'pass' ? 0 : 1);
+}
```

Comment:
I am still not sure what this flutter driver is but I saw a lot of plugins have it (https://github.com/flutter/plugins/tree/master/packages/google_sign_in/google_sign_in/test_driver)

It's also referenced in the docs of e2e https://pub.dev/packages/e2e#using-flutter-driver-to-run-tests

Running `flutter drive --driver=test_driver/purchases_flutter_e2e_test.dart ../test/purchases_flutter_e2e.dart` inside the `example` will run the e2e test in the emulator

Reply:
yes, and I think `e2e` is to run flutter tests in a simulator (using flutter drive), or something like that. Alternatively, someone could write UI tests on the example app in order to test the same